build.sh

```diff
@@ -95,17 +96,16 @@ echo
 echo "Adding L8e plugins"
 echo
 
+mkdir -p ${PLUGINS_DIR}
+
 for i in ${PLUGINS[@]}
 do
-    cd "${PLUGINS_DIR}/$i" && npm install && npm run build 
-    echo "Creating $i in plugins dir at project backend"
-    cd "${BUILD_DIR}"
-    mkdir -p "./plugins/$i"
-
-    echo "Copying plugins/$i/dist/main.js"
-    cp -R "${PLUGINS_DIR}/$i/dist/main.js" "./plugins/$i"
+    pushd "${PLUGINS_SRC_DIR}/$i"
+    npm run build
```

Comment:
Yeah, an argument to build would be convenient: `npm run build "${PLUGINS_SRC_DIR}/$i"`
Or even `npm run build "${PLUGINS_SRC_DIR}`